htdocs/core/modules/commande/doc/pdf_einstein.modules.php

```diff
@@ -251,9 +251,13 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
 				$dir = $conf->commande->multidir_output[$conf->entity];
 				$file = $dir."/SPECIMEN.pdf";
 			} else {
+				// Possibility to use suffix for proforma
+				if (isset($conf->global->PROFORMA_PDF_WITH_SUFFIX)){
+					$suffix = (GETPOST('model', 2)=='proforma') ? $conf->global->PROFORMA_PDF_WITH_SUFFIX : '';
+				}
 				$objectref = dol_sanitizeFileName($object->ref);
-				$dir = $conf->commande->multidir_output[$object->entity]."/".$objectref;
-				$file = $dir."/".$objectref.".pdf";
+				$dir = $conf->commande->dir_output . "/" . $objectref;
```

Comment:
Why replacing $conf->commande->multidir_output[$object->entity] with $conf->commande->dir_output ?

Reply:
i m sorry this is a wrong copy/paste from an old version of module_einstein